requirements.txt

```diff
@@ -1,6 +1,6 @@
 beautifulsoup4
 colorama==0.4.6
-dirtyjson==1.0.
```

Comment:
Why? It is unrelated to the Playwrite replacements. Separate PR. Even if it was related, delete, not comment.